docs/src/main/paradox/play-framework.md

```diff
@@ -49,8 +49,9 @@ again, in `application.conf`:
 
 @@snip[application.conf](/play-interop-test-scala/src/main/resources/application.conf) { #service-client-conf }
 
-See @ref[Client Configuration](client/configuration.md) for more information on the available options. If a client is generated and use that does not have an entry defined in `application.conf` it will fail with an exception
-when the client is injected. // FIXME depends on #271
+See @ref[Client Configuration](client/configuration.md) for more information on the available options. If the configuration
+is not present for that client and it is used by some other component, the application start will fail with an exception
+when injecting the client. // FIXME depends on #271
```

Comment:
Should we remove this "FIXME" note from the docs?

Reply:
I agree, let's either add a note to #271 to update these docs (or make sure it's commented out)

Reply:
Link to the issue since other places are doing that.